src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx

```diff
@@ -0,0 +1,214 @@
+        Display name
+      </td>
+
+      <td>
+        Optional. This is how the variable will display above the dashboard so that dashboard users know what the variable represents. If this is left blank, it will use the main name value. 
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Type
+      </td>
+
+      <td>
+There are three options: 
+* **Query**: You can write a query that will return a dynamic list of options used in the dropdown menu. For example, the following query would return a dynamic list of `country` values:
+  
+  ```
+  SELECT uniques(countryCode) FROM PageAction since 2 days ago
+  ```
+  For rules and tips on writing queries, see [Query-type variables](#query-variable-rules).
+* **List**: A list of comma-separated values that are used to populate the options in the dropdown menu. For example, you could manually define a list of `country` values using a list like: `ES, US, CA.`
+* **Text** field: Instead of a dropdown of values to choose from, this allows dashboard users to filter for whatever text they input. 
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Account
+      </td>
+
+      <td>
+        Only present for `query` type. For organizations with multiple accounts, this sets the account that is queried. 
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Default value
+      </td>
+
+      <td>
+        Optional. This is the default value that the dashboard will filter on. For example, if you used the `country` query above, you could input `ES` as the default value and the dashboard would automatically filter to that value. 
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+For an example of what a template variable for `country` values would look like, see [the screenshot at the top of this doc](#defined). 
+
+Once you've defined your template variable, you can move on to adding a template variable to your widgets. 
+
+### Step 2. Create widgets that use the variable [#create-widgets]
+
+Once you've configured a template variable, you'll need widgets on a dashboard that use the variable that you've defined in their query.
+
+Here's an example of creating a query that uses the `country` variable we mentioned in part 1: 
+
+```
+SELECT countryCode FROM PageAction WHERE countryCode = {{countryCode}}
+```
+
+A couple important points to note about using a variable in a query: 
+
+* The variable you defined goes inside the `{{ … }}` brackets. 
+* The variable generates a string value.  
+
+And here's an example of the resulting widget, on the right, with the `country` dropdown to the left. 
+
+<img
+  title="Country template variable example"
+  alt="Country template variable example"
+  src={dashboardsCountryVariableExample}
+/>
+
+When you're done a) defining a template variable and b) adding a widget that references that variable, you can verify it's working as expected by choosing different options from the template variable bar and seeing if the widget changes based on your selection. 
```

Comment:
Do we need the a) and b) here? It seems to read better without them, unless we want to change them to 1) and 2) to more directly reference the preceding steps. My gut says it would read better without them though. Just a thought.